scripts/set-default-contact/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "set-default-contact",
+  "version": "1.0.0",
+  "description": "Find users who's contact's places don't have a default contact, and set them to the user's contact",
+  "repository": {
+    "type": "git",
+    "url": "git@https://github.com/medic/cht-core"
+  },
+  "author": "Medic",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "nodemon": "^2.0.9",
+    "request": "^2.88.2",
+    "request-promise-native": "^1.0.8",
+    "node-yaml-config": "^0.0.6"
```

Comment:
I double checked and couldn't find any use of `nodemon` and `node-yaml-config`. If there is no need for them in this script, please `npm remove` them

Reply:
sweet! less is always more.